Resources/Prototypes/_White/Decals/crayons.yml

```
@@ -74,3 +74,368 @@
     sprite: WhiteDream/BloodCult/Entities/Runes/regular.rsi
     state: blood_boil
+
+
+- type: decal
+  id: а
+  parent: Alphabet
+  tags: ["crayon", "crayon-2-alphanum"]
+  defaultCleanable: true
+  defaultCustomColor: true
+  defaultSnap: false
+  sprite: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)